src/Main.java

```diff
@@ -0,0 +1,23 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by avery on 12/6/16.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+        ArrayList<Character> characters = new ArrayList();
+        characters.add(new Ranger("Avery", 1.0));
+        characters.add(new Ranger("Alex", 1.0));
+        while(characters.get(0).isAlive()&&characters.get(1).isAlive()) {
+            characters.get(0).attack(characters.get(characters.get(0).pickTarget(characters)));
+            characters.get(1).attack(characters.get(characters.get(1).pickTarget(characters)));
+        }
```

Comment:
This should be a loop. It needs to loop through every character and do all of this. It should remove characters as they die.